microhapulator/tests/data/def/nocov-offsets.tsv

```diff
@@ -0,0 +1,16 @@
+Marker	Offset
+mh02USC-2qE	151
+mh02USC-2qE	170
+mh02USC-2qE	174
+mh03USC-3qB	143
```

Comment:
The four markers in the BAM file are a subset of the 5 markers in this definition file.